miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditExperimentController.java

```diff
@@ -227,7 +231,7 @@ public String processSubmit(@ModelAttribute("experiment") Experiment experiment,
         throw new SecurityException("Permission denied.");
       }
       experiment.setLastModifier(user);
-      requestManager.saveExperiment(experiment);
+      experimentService.saveExperiment(experiment);
```

Comment:
looking back at these methods, `experimentService.saveExperiment(experiment)` could simply be `experimentService.save(experiment)`, likewise with `getExperimentById(id)`.
It seems the convention is to be more verbose, especially with id columns in the db!  I'd like to move away from that at some point.